modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,80 @@
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users were unable to include context for notifications, in the `NotificationsConfiguration` custom resource (CR). With this update, users can now add notifications by utilizing the `context` field within the `NotificationsConfiguration` (CR). This field enabled users to add context in the `NotificationsConfiguration` (CR) and can also be utilized to establish shared context across all templates defined in the `NotificationsConfiguration` (CR). link:https://issues.redhat.com/browse/GITOPS-4304[GITOPS-4304]
+
++
+Example for `context`
++
+[source,yaml]
+----
+spec:
+  context:
+    key1: value1
+    key2: value2
+  templates:
+    template.app-created: |-
+      email:
+        subject: Application {{.app.metadata.name}} has been created.
+      message: Application {{.app.metadata.name}} has been created.
+      teams:
+        title: Application {{.app.metadata.name}} has been created.
+    template.app-deleted: |-
+     ...
+     ...
+----
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to login to the Argo CD Web Console UI due to an error indicating an invalid redirect URI in the Dex configuration. To resolve this issue, users had to restart the {gitops-title} Operator pod along with the Server and Dex pods of Argo CD. With this update, users can now log in to the Argo CD Web Console without facing any errors once the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+* Before this update, users were unable to log in to the Argo CD Web Console UI when there is a delay in the availability of `openshift-gitops` route during processing of an instance of the Argo CD custom resource by the {gitops-title} Operator. An error message is displayed indicating an invalid redirect URI in the Dex configuration. To resolve this issue, users had to restart the {gitops-title} Operator pod along with the Server and Dex pods of Argo CD. With this update, users can now log in to the Argo CD Web Console without facing any errors. link:https://issues.redhat.com/browse/GITOPS-3736[GITOPS-3736]
+
+* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases.
```

Comment:
```suggestion
* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} web console. This issue has been observed from {gitops-title} 1.10 and later releases.
```

Reply:
The part of the fix is missing in this release note: "With this fix/this update fixes"
Is it fixed or is this rather a known issue?

Reply:
It is a fixed issue and I have included the updates received from the SME.